### ISSUE TITLE
filename aux.py prevents installation on windows

### ISSUE THREAD

**Issue by poopipe** [leomariga/pyRANSAC-3D#11]
I've just had some nasty issues installing on windows(10) due to 'aux' being a reserved word wrt to filenames (aux.py) , 

This issue prevents installation with pip, and leaves the user unable to open, rename, or delete the file - making it effectively impossible to clean up afterwards.

**Comment by leomariga** [comment]
Thank you for the issue. I'm working on a solution for this as soon as possible.

**Comment by leomariga** [comment]
Fix on #12 👍 

```
Collecting pyransac3d
  Downloading pyransac3d-0.5.0-py3-none-any.whl (18 kB)
Installing collected packages: pyransac3d
Successfully installed pyransac3d-0.5.0
```
